.github/workflows/_release.yml

```
@@ -306,7 +306,19 @@ jobs:
       contents: write
     env:
       TAG: ${{ needs.release.outputs.tag }}
+      VERSION: ${{ needs.release.outputs.version }}
```

actionlint (1.7.12)

[error] property "version" is not defined in object type {tag: string}

(expression)